src/NHSD.BuyingCatalogue.Database/Tables/CataloguePrice.sql

```diff
@@ -0,0 +1,17 @@
+﻿CREATE TABLE dbo.CataloguePrice
+(
+    CataloguePriceId INT IDENTITY(1,1) NOT NULL,
+    CatalogueItemId NVARCHAR(14) NOT NULL,
+    ProvisioningTypeId INT NOT NULL,
+    CataloguePriceTypeId INT NOT NULL,
+    PricingUnitId UNIQUEIDENTIFIER NOT NULL,
+    TimeUnitId INT NULL,
+    CurrencyCode NVARCHAR(3) NOT NULL,
+    LastUpdated DATETIME2(7) NOT NULL
```

Comment:
There should be a comma at the end of here

Reply:
The project does compile without the comma but added them.